_config.yml

```diff
@@ -87,6 +87,9 @@ collections:
   th: # Thai translations
     output: true
     permalink: /:collection/:path.html
+  uk: # Ukrainian translations
```

Comment:
Why 'uk' and not 'ua'? 
Usually 'uk' uses for United Kingdom.

Reply:
It's a language code and might be a little bit confusing.
Also, there are country codes and there are pairs like `en-us`, en-uk`, `en-ca`, fr-ca` and `uk-ua`
https://www.loc.gov/standards/iso639-2/php/code_list.php